nautobot/core/views/mixins.py

```diff
@@ -92,7 +92,13 @@ def get_required_permission(self):
         Obtain the permissions needed to perform certain actions on a model.
         """
         queryset = self.get_queryset()
-        return self.get_permissions_for_model(queryset.model, [PERMISSIONS_ACTION_MAP[self.action]])
+        try:
+            permissions = [PERMISSIONS_ACTION_MAP[self.action]]
+        except KeyError:
+            messages.error(
+                "This action is not permitted. Please use the buttons at the bottom of the table for Bulk Delete and Bulk Update"
+            )
```

Comment:
Double-check if I'm wrong on this, but pretty sure this will error out, I think it needs to be

```suggestion
            messages.error(
                self.request,
                "This action is not permitted. Please use the buttons at the bottom of the table for Bulk Delete and Bulk Update"
            )
```